hybridse/src/udf/default_udf_library.cc

```diff
@@ -272,6 +272,47 @@ struct AvgUdafDef {
     }
 };
 
+template <typename T>
+struct StdUdafDef {
+    using ContainerT = std::pair<std::vector<T>, double>;
+    void operator()(UdafRegistryHelper& helper) {  // NOLINT
+        std::string suffix = ".opaque_std_pair_std_vector_double_" + DataTypeTrait<T>::to_string();
+        helper.templates<Nullable<double>, Opaque<ContainerT>, Nullable<T>>()
+            .init("std_init" + suffix, Init)
+            .update("std_update" + suffix, Update)
+            .output("std_output" + suffix, reinterpret_cast<void *>(Output), true);
+    }
+
+    static void Init(ContainerT* ptr) {
+        new (ptr) ContainerT(std::vector<T>(), 0.0);
+    }
+
+    static ContainerT* Update(ContainerT* ptr, T t, bool is_null) {
+        if (!is_null) {
+            ptr->first.emplace_back(t);
+            ptr->second += t;
+        }
+        return ptr;
+    }
+
+    static void Output(ContainerT* ptr, double* ret, bool* is_null) {
+        if (ptr->first.size() == 0) {
+            *is_null = true;
+        } else {
+            size_t cnt = ptr->first.size();
+            double avg = ptr->second / cnt;
+            double stddev = 0;
+            for (size_t i = 0; i < cnt; i++) {
+                stddev += std::pow(ptr->first[i] - avg, 2);
+            }
+            stddev = std::sqrt(stddev / cnt);
```

Comment:
Sample Standard Deviation should divide by `cnt -1` ?

Reply:
oh. the formula I previously gave in the PR text is wrong. The correct one should be 
![image](https://user-images.githubusercontent.com/7508922/218017231-71568708-b3a6-41da-b6bb-04a1f3fa2bd5.png)
 (population standard deviation), which is also the definition in MySQL.